piglot/solver/abaqus/solver.py

```diff
@@ -116,19 +124,21 @@ def _run_case(self, values: np.ndarray, case: Case, tmp_dir: str) -> CaseResult:
             check=False
         )
 
-        variables = self._post_proc_variables(input_data)
+        variables = self._post_proc_variables(input_data, list(case.fields.values())[0])
         python_script = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'reader.py')
 
         run_odb = subprocess.run(
             [self.abaqus_bin, 'viewer', f"noGUI={python_script}", "--",
              f"input_file={variables['input_file']}", "--",
              f"job_name={variables['job_name']}", "--",
              f"step_name={variables['step_name']}", "--",
-             f"instance_name={variables['instance_name']}"],
+             f"instance_name={variables['instance_name']}", "--",
+             f"set_name={variables['set_name']}", "--",
+             f"field={variables['field']}", "--",
+             f"x_field={variables['x_field']}"],
             cwd=tmp_dir,
             shell=False,
             stdout=subprocess.DEVNULL,
-            stderr=subprocess.DEVNULL,
```

Comment:
I deleted that because if in the `reader.py` file a error is raised I want it to appear in the terminal to be more helpful for the user. For example if the user is running a infinitesimal case and request for logarithmic strains it raises a error warning the user. If the output is suppressed the user doesn't know why it is giving the error.

Reply:
I'm not comfortable with disabling this suppression at this stage specifically for a single solver. I agree that we should handle these scenarios, but the solution should be solver-agnostic (likely by piping to appropriate streams and then dumping them to the output directory). I can work on implementing this feature in an upcoming PR.

Reply:
I understand. I will keep this suppression for now.